### PR TITLE
setup documentation config

### PR DESCRIPTION
**Pull request by memo330179**
Build html with `sphinx-build -b html sourcedir builddir`
Dependencies `sphinx`, `sphinxcontrib-napoleon`

### CODE REVIEW

**Comment by Jessonsotoventura** [comment]
The documentation provided is only generating an index of the modules, and not providing me with the description of what each module does. Is this expected?

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = 'Submitted'
+copyright = '2018, Jesson Soto Ventura, Guillermo Ramos Macias'
```
Comment (Jessonsotoventura) [inline docs/source/conf.py]: I think we should steer away from attributing all the documentation to just ourselves. There will be others that also work on it, so can we change this to Berea College Student Software Development Team or BCSD (Berea College Student Developers), I think the latter is better, seeing as how we want this to be a project for all CS students not just the Development team

File: docs/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    #
+    # 'figure_align': 'htbp',
+}
+
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title,
+#  author, documentclass [howto, manual, or own class]).
+latex_documents = [
+    (master_doc, 'Submitted.tex', 'Submitted Documentation',
+     'Jesson Soto Ventura, Guillermo Ramos Macias', 'manual'),
```
Comment (Jessonsotoventura) [inline docs/source/conf.py]: I think we should steer away from attributing all the documentation to just ourselves. There will be others that also work on it, so can we change this to Berea College Student Software Development Team or BCSD (Berea College Student Developers), I think the latter is better, seeing as how we want this to be a project for all CS students not just the Development team

**Comment by memo330179** [comment]
@Jessonsotoventura Made those changes thanks for catching that.

**Comment by Jessonsotoventura** [comment]
When building I get the following two warnings, and still not description of what the functions do:
```
loading pickled environment... not yet created
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 3 source files that are out of date
updating environment: 3 added, 0 changed, 0 removed
reading sources... [100%] modules
looking for now-outdated files... none found
pickling environment... done
checking consistency... /Users/jesson/Development/Github/Project_ART/docs/source/modules.rst: WARNING: document isn't included in any toctree
done
preparing documents... done
writing output... [100%] modules
generating indices... genindex py-modindex
writing additional pages... search
copying static files... WARNING: html_static_path entry '/Users/jesson/Development/Github/Project_ART/docs/source/_static' does not exist
done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded, 2 warnings.
```
screen shot 2018-01-24 at 9 52 27 pm
^ I know these functions have docstrings but nothing is showing

**Comment by Jessonsotoventura** [comment]
Can you update this to include the models and queries folder?
If possible it might be a good idea to have it include all the controllers too.
We might not have documentation for them, but at least we will know what functions exist.

**Comment by sheggen** [comment]
Not sure what to do with this pull request. Either merge it (if it's important) or can it, please.

**Pull request closed without merging**